src/Training.AirBnb.Clone.Backend/Backend Project/Infrastructure/Services/LocationServices/CountryService.cs

```diff
@@ -87,8 +87,8 @@ private bool IsValidCountryDailingCode(Country country)
                 return false;
 
             if (!(country.CountryDialingCode[0].Equals("+")
-                || country.CountryDialingCode.Length > 1
-                || country.CountryDialingCode.Length < 5))
+                || country.CountryDialingCode.Length < 2
+                || country.CountryDialingCode.Length > 5))
                 return false;
```

Comment:
ifdagi undovni turishi to'grimi xozir logika noto'g'ri ishlab qolmayabdimi
o'zi xato xolatga tekshirilgan o'shani yana inkor qilib true bo'lsa false qaytarabmiz 
logika teskari ishlab qolmaganmi